CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # 1.4.0 (2022-08-26)
 
+## What's new in 1.4?
+
+The 1.4 version is the latest long-term support (LTS) version of IoT Edge. It will be serviced with fixes for regressions and critical security issues through November 12, 2024. In addition to long-term servicing, it includes the following improvements.
+* Automatic image cleanup of unused Docker images
+* Ability to pass a custom json payload to DPS on provisioning
+* Option to download all modules in a deployment before (re)starting any
+* Use of the TCG TPM2 Software Stack which enables TPM hierarchy authorization values, specifying the TPM index at which to persist the DPS authentication key, and accommodating more TPM configurations (doc)
```

Comment:
Is "(doc)" a placeholder for a doc link?

Reply:
Ah, the hyperlinks didn't transfer across.  the "(doc)" should point to `https://github.com/Azure/iotedge/blob/897aed8c5573e8cad4b602e5a1298bdc64cd28b4/edgelet/contrib/config/linux/template.toml#L262-L288`